scripts/seed.ts

```diff
@@ -191,9 +191,8 @@ async function createPlatformAndSetupUser({
         permissions: 1023,
         areEmailsEnabled: true,
         organizationId: team.id,
-        id: "clxyyy21o0003sbk7yw5z6tzg",
-        secret:
-          "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJuYW1lIjoiQWNtZSAiLCJwZXJtaXNzaW9ucyI6MTAyMywicmVkaXJlY3RVcmlzIjpbImh0dHA6Ly9sb2NhbGhvc3Q6NDMyMSJdLCJib29raW5nUmVkaXJlY3RVcmkiOiIiLCJib29raW5nQ2FuY2VsUmVkaXJlY3RVcmkiOiIiLCJib29raW5nUmVzY2hlZHVsZVJlZGlyZWN0VXJpIjoiIiwiYXJlRW1haWxzRW5hYmxlZCI6dHJ1ZSwiaWF0IjoxNzE5NTk1ODA4fQ.L5_jSS14fcKLCD_9_DAOgtGd6lUSZlU5CEpCPaPt41I",
+        id: process.env.SEED_PLATFORM_OAUTH_CLIENT_ID || "",
```

Comment:
If SEED_PLATFORM_OAUTH_CLIENT_ID and SEED_PLATFORM_OAUTH_CLIENT_SECRET are not defined i don't see a reason to call `await prisma.platformOAuthClient.create` at all. I would call it only if they are defined and if not then console.log warning that id and secret are missing and that platform oauth client will not be seeded.